From fee973e3266ebc3ad4aafdd0d88d7b8010cb9d54 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Fri, 29 May 2026 16:49:18 -0700
Subject: [PATCH] Default CPU compatibility to "normal" on x86_64

Remove the x86_64-specific default of memory and use normal for all targets
in core options initialization.
---
 libretro/libretro-core.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/libretro/libretro-core.c b/libretro/libretro-core.c
index d27ffe25..f05a0932 100644
--- a/libretro/libretro-core.c
+++ b/libretro/libretro-core.c
@@ -1107,11 +1107,7 @@ static void retro_set_core_options()
             { "exact", "Cycle-exact (Full)" },
             { NULL, NULL },
          },
-#if defined(__x86_64__)
-         "exact"
-#else
          "normal"
-#endif
       },
       {
          "puae_autoloadfastforward",
-- 
2.50.1 (Apple Git-155)

